README.md

```diff
@@ -34,6 +34,12 @@ You can specify options in `.vcmrc`
 ```js
 {
   "types": ["feat", "fix", "docs", "style", "refactor", "perf", "test", "chore", "revert"], // default
+  "scopes": {
```

Comment:
I would maybe have this a singular. `types` is plural as the actual value is an array. But in this case I see this more as config for `scope` property of the commit message. What do you think?